index.html

```diff
@@ -173,6 +179,64 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<div class="green">
+
+<h2>Tips for achieving consensus</h2>
+
+<p>During the discussion of a proposal any aspect may be discussed. Consensus is given as an indicator of the current stage. Delegates should openly give feedback on proposals, and especially for a proposal for stage advancement where the concern is relevant to the stage. Delegates should raise their concerns early and asynchronously, in order to help the champion resolve any issues.
+
+<p>A delegate may pose a constraint as necessary for advancement. A constraint refers to an desired property of the proposal, accompanied by a rationale. We encourage this to also be done asynchronously in issues, and in incubator calls, as well as in plenary. In this situation, the delegate should expect to work with the champion and other delegates during earlier stages to incorporate their constraint into the solution, and to consider different possible tradeoffs. In general, the earlier a constraint is raised, the better.
+
+<p>Frequently, many different conflicting constraints are posited about proposals, and the committee collectively may make tradeoffs selecting a particular design even though it compromises one or more constraints.
+
+<p>Given that consensus on Stage 3 means "the solution is complete", i.e., all open design issues have been resolved including anticipated implementation and ecosystem compatibility issues. All TC39 participants should validate the design of proposals they care about before granting Stage 3 consensus. Stage 3 proposals which have fulfilled the acceptance criteria for Stage 4 may not be withheld from advancement unless the issue raised is related to implementation experience or identifies a problem or information which has not previously been discussed by the committee. The intention is to allow implementers to invest in implementations, and maintain the significance of stage 3 in the process.
+
+</div>
+<div class="green">
+
+<h2>In cases where the committee does not come to consensus</h2>
+
+<p>The committee may come to a point where consensus is not reached in committee regarding the feature. In this case, the committee must record a good description of why a proposal did not advance. This should be done both in the meeting notes and within an issue in the proposal's tracker, but not limited to those. This allows us to understand issues in the proposal and similar proposals in a coherent way.
+
+<p>There are two forms that this typically takes, the first is the violation of a constraint and the other is colloquially known as a block.
```

Comment:
I'm concerned that "typically" has the misleading connotation that other forms are too rare to worry about, downplaying the possibility of the committee finding a proposal introducing complexity not paying for itself or simply being undesirable. I'd prefer to replace "typically" by "sometimes" and provide an indication to emphasize that these two forms are not the only ones.

Reply:
I have replaced "typically" with "sometimes" as suggested. Let me know if there are other wording concerns.